src/applications/dependents/view-dependents/components/ViewDependentsHeader/ViewDependentsHeaderV2.jsx

```diff
@@ -15,13 +15,27 @@ const CALLSTATUS = {
   skip: 'skipped',
 };
 
+function getIsWarningHidden() {
+  const rawStoredDate = localStorage.getItem('viewDependentsWarningClosedAt');
+  if (!rawStoredDate) {
+    return false;
+  }
+
+  const dateClosed = new Date(rawStoredDate);
```

Comment:
The `rawStoredDate` is always going to be a string (localStorage restriction), so using `new Date('any string')` is going to return an invalid date.

Reply:
oops, you're storing it as an ISO date... in my head I was thinking you used `Date.now()`